docs/content/quickstart/parameters.md

```diff
@@ -0,0 +1,147 @@
+* Deployment Region: Let the user specify which region, such as eastus-1, where the application should be deployed.
+* Helm Release Name: A bundle that uses Helm often will define a parameter that allows the user to set the release name for the Helm release.
+
+For optional parameters, bundles set a default value that is used when the user does not specify a value for the parameter.
+
+Let's look at a bundle with parameters:
+
+```console
+$ porter explain --reference getporter/hello-llama:v0.1.1
+Name: hello-llama
+Description: An example Porter bundle with parameters
+Version: 0.1.0
+Porter Version: v0.38.1-32-gb76f5c1c
+
+No credentials defined
+
+Parameters:
+Name   Description                           Type     Default   Required   Applies To
+name   Name of to whom we should say hello   string   llama     false      All Actions
+
+No outputs defined
+
+No custom actions defined
+
+No dependencies defined
+```
+
+The output tells us that the bundle has one optional string parameter, name, which defaults to "llama".
+The name parameter applies to "All Actions" meaning that this parameter can be specified with every action that the bundle defines.
+Since it says that no custom actions are defined, the bundle only supports the built-in actions of install, upgrade, and uninstall.
+
+## Use the default parameter values
+
+
+First install the bundle and do not specify any parameters so that you can observe how the bundle installs without any customization.
+
+```console
+$ porter install hello-llama --reference getporter/hello-llama:v0.1.1
+installing hello-llama...
+executing install action from hello-llama (installation: hello-llama)
+Hello, llama
+execution completed successfully!
+```
+
+The bundle printed "Hello, llama" using the default value for the name parameter.
+
+## Specify a parameter with a flag
+Next upgrade the installation and change the name parameter to another value.
+Parameters are specified with the \--param flag: 
+
+```console
+$ porter upgrade hello-llama --param name=Porter
+upgrading hello-llama...
+executing upgrade action from hello-llama (installation: hello-llama)
+Porter 2.0
+execution completed successfully!
+```
+
+## Create a Parameter Set
+Setting a parameter value individually on the command line with the \--param flag works well when trying out a bundle.
+When working with a bundle often though, remembering and typing out every parameter values every time is error prone.
+Parameter Sets store a set of parameter values to use with a bundle.
+Even more powerful, the value of the parameters can come from the following sources:
+
+* hard-coded value
+* environment variable
+* file
+* command output
+* secret
+
+Some parameters may be sensitive, for example a database connection string or oauth token.
+For improved security, and to limit exposure of sensitive values, it is recommended that you source those parameter values from a secret store such as Hashicorp Vault or Azure Key Vault.
```

Comment:
For improved security, and to limit exposure of sensitive values, it is recommended that you source sensitive parameter values from a secret store such as HashiCorp Vault, Azure Key Vault or Google Secret Manager.

Reply:
I can't mention google because no one has made a plugin for that yet.

Reply:
AHHH. just cut the Google Secret Manager out and keep the "sensitive parameter values" (instead of those)